motion_predict/include/motion_predict/motion_predict.hpp

```diff
@@ -35,16 +35,16 @@ namespace cv{
     \param  input is the current value of the process noise.
     \param  process_noise_max is the maxium process noise of the system
     */
-
     double Mapping(const double input,const double process_noise_max);
 
     /*!
     \brief  predictState is used to predict future state.
     \param  pose is position and orientation (m).
     \param  twist is velocity (m/s).
     \param  delta_t time predicted into the future (sec).
+    \note twist.linear vector is expected to be in the map frame and used as the yaw for future prediction.
+          pose.orientation is not meaningfully used at the moment except passing it on.
```

Comment:
Nit: Might be more concise to say that `pose` and `twist` are expected in the map frame.

Reply:
edited